.landscaper/gardenlet/blueprint.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+jsonSchema: "https://json-schema.org/draft/2019-09/schema" # required
+
+# imports are filled via the Installation CRD specifying an import section.
+imports:
+- name: runtimeCluster
+  targetType: landscaper.gardener.cloud/kubernetes-cluster
+- name: gardenCluster
+  targetType: landscaper.gardener.cloud/kubernetes-cluster
+- name: imageVectorOverwrite
+  schema:
+    type: string
+- name: componentImageVectorOverwrites
+  schema:
+    type: string
+- name: seedBackup
+  schema:
+    type: object
+    properties:
+      name:
```

Comment:
"name" is not very clear - is this the provider type or the secret name or what?